src/vos/vos_policy.c

```diff
@@ -0,0 +1,71 @@
+/**
+ * (C) Copyright 2016-2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * Object placement policy
+ *  Policy definitions
+ *
+ * Author: Krzysztof Majzerowicz-Jaszcz (krzysztof.majzerowicz-jaszcz@intel.com)
+ */
+
+#include <daos/policy.h>
+#include "vos_policy.h"
+
+
+/* policy functions definitions */
+
+/* default policy - former "vos_media_select" function */
+static daos_media_type_t
+policy_default(struct vos_pool *pool, daos_iod_type_t type, daos_size_t size)
```

Comment:
ah, I see, this is named 'default' policy. I think current policy should be merged into io size policy, right?

Reply:
Please get rid of this 'default' policy, and use 'io size' policy as default policy.